Sources/ForageSDK/ForageSDK.swift

```diff
@@ -43,13 +43,15 @@ public class ForageSDK {
         VGSCollectLogger.shared.disableAllLoggers()
        
         self.service = LiveForageService()
+        if ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] == nil {
```

Comment:
looks good! Can we assign this to a var `isUnitTesting`?
and then do `if !isUnitTesting { ... }`

I'm curious whether this happens when we run the [iOS SDK e2e tests](https://github.com/teamforage/mobile-qa-tests) locally. I feel like it isn't, but I can check once  this PR gets a ✅

Reply:
79a03ac028f64d664c6c33149ff84c9cf1ee0527